bin/deploy

```diff
@@ -13,7 +13,10 @@ end
 
 env = ARGV.shift
 image = ARGV.shift
-cluster = if env == "sandbox"
+
+aws_account_alias = JSON.parse(`aws iam list-account-aliases --output json --query "AccountAliases[0]"`)
+
+cluster = if env == "sandbox" or aws_account_alias == "idseq-prod"
             "idseq-#{env}-ecs"
           else
             # TODO(2020-01-23): Rename staging/prod clusters to new format (IDSEQ-2155).
```

Comment:
nit: should we update this comment, since prod is just going to be removed.

Reply:
Maybe, but it is worth pointing out the comment still valid. `prod` still there, and `staging` is not following the new format either

Reply:
What meant is that we should change it to mention staging only. Prod should simply be removed after (we should not spend time renaming it anymore)